src/Core/Security/Application/ProviderConfiguration/OpenId/UseCase/UpdateOpenIdConfiguration/UpdateOpenIdConfiguration.php

```diff
@@ -91,4 +114,48 @@ private function getContactTemplateOrFail(?array $contactTemplateFromRequest): ?
 
         return $contactTemplate;
     }
+
+    /**
+     * Get Contact Group or throw an Exception
+     *
+     * @param integer|null $contactGroupId
+     * @return ContactGroup|null
+     * @throws \Throwable|OpenIdConfigurationException
+     */
+    private function getContactGroupOrFail(?int $contactGroupId): ?ContactGroup
+    {
+        if ($contactGroupId === null) {
+            return null;
+        }
```

Comment:
I think this check should be remove from this method because it's not consistent with the method's name
`$contactGroupId` should not be null